staging/src/k8s.io/apiserver/pkg/apis/apiserver/v1alpha1/types.go

```diff
@@ -167,3 +167,41 @@ type TracingConfiguration struct {
 	// Defaults to 0.
 	SamplingRatePerMillion *int32 `json:"samplingRatePerMillion,omitempty" protobuf:"varint,2,opt,name=samplingRatePerMillion"`
 }
+
+// FlowControlConfig holds priority and fairness parameters.
+type FlowControlConfig struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// WorkEstimatorConfig holds work estimator parameter values.
+	*WorkEstimatorConfig `json:"workEstimatorConfig,omitempty"`
+}
+
+// WorkEstimatorConfig holds work estimator parameters.
+type WorkEstimatorConfig struct {
+	ListWorkEstimatorConfig     *ListWorkEstimatorConfig     `json:"listWorkEstimatorConfig,omitempty"`
+	MutatingWorkEstimatorConfig *MutatingWorkEstimatorConfig `json:"mutatingWorkEstimatorConfig,omitempty"`
+
+	// MinimumSeats is the minimum number of seats a request must occupy.
+	MinimumSeats uint `json:"minimumSeats,omitempty"`
```

Comment:
I do not think we want this to be a configurable variable.  It was OK as a constant in the code; I would have been happy with not even defining a symbolic constant for it.

Reply:
Yeah - for minimum I agree it's probably not needed.

Reply:
Removed.